dockers/docker-syncd-mlnx/start.sh

```diff
@@ -8,6 +8,10 @@ function clean_up {
 
 trap clean_up SIGTERM SIGKILL
 
+# fw-upgrade will exit if firmware was actually upgraded or if some error
+# occures
+. mlnx-fw-upgrade.sh
```

Comment:
It is actually a trick to exit directly from mlnx-fw-upgrade.sh. Please add your comment as code comment.

BTW, syncd service to restart ==  restart sxdkernel driver?